.nvmrc

```diff
@@ -1 +1 @@
-v10
+v16
```

Comment:
Did you intend to set this to 16 and not 14?

Reply:
good catch on the inconsistency. I could go either way here - do you have a preference?

Reply:
I would lean on going with the standards established here: https://github.com/MetaMask/metamask-module-template

Once we bump in that, that feels like approval to do it in every other package 👍

Reply:
Thanks for bringing that to my attention - in light of that let's go with 14 here for now.